src/js/routes/Ballot/Ballot.jsx

```diff
@@ -23,7 +24,6 @@ import SupportStore from "../../stores/SupportStore";
 import VoterActions from "../../actions/VoterActions";
 import VoterConstants from "../../constants/VoterConstants";
 import VoterStore from "../../stores/VoterStore";
-import EditAddress from "../../components/Widgets/EditAddress";
 
 
 const web_app_config = require("../../config");
```

Comment:
Hi @SailingSteve, did you intend to leave the prior modals that you refactored in this version of the Ballot.jsx file?